### DISCUSSION TITLE
Under utilization of 2x 2TB NVME?

### DISCUSSION THREAD

**Discussion by DroptheHammer** [ericaltendorf/plotman#534] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/79561845/118569273-712bb300-b72e-11eb-8bbc-4a38f8d108d9.png)

It seems after 8 hours or so that my drives aren't getting anywhere near the recommended 6-7 concurrent plots per 2TB NVME and I think I'm running close to recommended settings...

Any tips on the below?  

I have..

```
 tmpdir_stagger_phase_major: 2
 tmpdir_stagger_phase_minor: 1
 tmpdir_stagger_phase_limit: 1
tmpdir_max_jobs: 7
global_max_jobs: 14
global_stagger_m: 40
polling_time_s: 20
```

**Comment by skeg43** [comment, marked as answer]
I am not an expert, but I had the same problem. I increased the stagger phase limit. Right now with it set at 1 you can only have 1 job of that type per phase. etc 1 job in phase 1, 1 job in phase 2. I changed mine to 2 and now I run 2 in each phase 1, 2 ,3.

**Comment by cyperbg** [comment]
tmpdir_stagger_phase_limit: 1
This is your problem. Increase it to 3

Now you can only have 1 active job that is below phase 2:1 per drive.
